### PR TITLE
Add modalities tag to blog

### PR DESCRIPTION
**Pull request by mishig25**
Add 4 modalities tags (`nlp, speech, vision, rl`) (technically, I guess, it is 3 modalities + rl) & tag relevant blog posts
(as requested by @patrickvonplaten)

Screenshot 2022-02-07 at 09 41 58

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -364,6 +394,7 @@
   date: November 15, 2021
   tags:
   - guide
+  - nlp
```
Comment (patrickvonplaten) [inline _blog.yml:397]:
```suggestion
  - speech
```

File: _blog.yml [hunk 2]
```diff
@@ -467,6 +506,7 @@
   date: January 21, 2022
   tags:
   -  open-source-collab
+  -  rl
 
 - local: searching-the-hub
   title: "Supercharged Searching on the Hugging Face Hub"
```
Comment (patrickvonplaten) [inline _blog.yml:512]: further below at line 525 (can't comment there :-/) - could we also add the speech tag: ?

**Comment by patrickvonplaten** [comment]
Awesome thanks a lot!

File: _blog.yml (hunk 2 above)
Comment (mishig25) [reply _blog.yml:512]: done

File: _blog.yml [hunk 3]
```diff
@@ -483,11 +523,13 @@
   tags:
   - guide
   - research
+  - speech
```
Comment (osanseviero) [inline _blog.yml:526]: cannot comment, but could we add a new line above asr-chunking ?(518)

**Review by osanseviero — APPROVED** [review]
Thanks for the PR!

**Comment by julien-c** [comment]
can we use the same modalities names as in https://github.com/huggingface/huggingface_hub/blob/a217a7e04d4c6785e36fe193ee2576be95815157/js/src/lib/interfaces/Types.ts#L62-L67 for consistency? (and add missing ones there if needed)

More generally let's try to promote naming consistency accross the huggingface platform, going forward :)

**Comment by julien-c** [comment]
do we still want to merge this BTW? 🙏

**Comment by mishig25** [comment]
I've added the modalities tags to newer blog posts as well. 

> can we use the same modalities names as in https://github.com/huggingface/huggingface_hub/blob/a217a7e04d4c6785e36fe193ee2576be95815157/js/src/lib/interfaces/Types.ts#L62-L67 for consistency? (and add missing ones there if needed)

Implemented naming consistency based on below `const`.
```
export const MODALITIES = [
	"nlp",
	"audio",
	"cv",
	"other",
] as const;
```

For reinforcement learning (which does not exist in const MODALITIES), I've chose `rl`. wdyt @simoninithomas ?

**Comment by julien-c** [comment]
you can maybe add `rl` to modalities now, or does it not make sense @osanseviero?

**Comment by mishig25** [comment]
Here is the hub-docs [PR](https://github.com/huggingface/hub-docs/pull/90) in case we want to add _rl to modalities_

**Comment by osanseviero** [comment]
I think you can merge [rl_modality](https://github.com/huggingface/hub-docs/tree/rl_modality) to this branch and then merge all into main.

**Pull request merged**